### DISCUSSION TITLE
Unknown parameters on equation and boundary conditions

### DISCUSSION THREAD

**Discussion by hei6775** [lululxvi/deepxde#798] [Q&A]
Hello, I have one question about the unknown parameters on equation and boundary conditions. Yes, both equation parameter $D$ and boundary conditions $B$ are unknown. The PDE is:
$$\frac{\partial c}{\partial t} =  D\frac{\partial^2 c }{\partial x^2}$$
The BC is $D\frac{\partial c}{\partial t}=B$ and IC is $c(x,0)=1$

from my experiment it seems that the loss does not decrease. Can deepxde solve this similar problem?

**Comment by lululxvi** [comment]
It is tricky to implement an inverse problem with unknown in BC. You can assume the BC condition is another PDE only defined in the boundary locations.
